marquez_client/client.py

```diff
@@ -99,34 +110,40 @@ def create_job(self, job_name, location, input_dataset_urns=None,
         if description:
             payload['description'] = description
 
-        return self._put(
+        response = self._put(
             self._url('/namespaces/{0}/jobs/{1}', namespace_name, job_name),
             payload=payload
         )
 
+        return Job.from_response(response)
+
     def get_job(self, job_name, namespace_name=None):
         if not job_name:
             raise ValueError('job_name must not be None')
 
         if not namespace_name:
             namespace_name = self._namespace_name
 
-        return self._get(
+        response = self._get(
             self._url('/namespaces/{0}/jobs/{1}', namespace_name, job_name)
         )
 
+        return Job.from_response(response)
+
     def list_jobs(self, limit=None, offset=None, namespace_name=None):
         if not namespace_name:
             namespace_name = self._namespace_name
 
-        return self._get(
+        responses = self._get(
             self._url('/namespaces/{0}/jobs', namespace_name),
             params={
                 'limit': limit,
                 'offset': offset
             }
         )
 
+        return [Job.from_response(response) for response in responses]
```

Comment:
The response had been updated for runs to introduce the top level node `runs`, see https://github.com/MarquezProject/marquez/pull/594. We'll want to update how we process a runs list response.